code/stable/projectile/projectile_u_p_nol_u_wi_v_d/src/cpp/Projectile.cpp

```diff
@@ -33,39 +33,19 @@ int main(int argc, const char *argv[]) {
     double p_target;
     get_input(filename, v_launch, theta, p_target);
     input_constraints(v_launch, theta, p_target);
-    double t_flight = func_t_flight(v_launch, theta, g);
-    double p_land = func_p_land(v_launch, theta, g);
-    double d_offset = func_d_offset(p_target, p_land);
-    string s = func_s(p_target, epsilon, d_offset);
+    // flight duration: the time when the projectile lands (s)
```

Comment:
Why have the comments started appearing here? [Don't get me wrong, this is a great *option* to have!] But we should have a version without them too.

Reply:
> When comments are turned on, I think a version of these comments should go above the inlined code.

These are meant to be the inlined code comments. It's just a weaker variation of the ones that would go above the function-generating counterpart. I don't think we want the full commentary, do we?